addons/editor-dark-mode/experimental_editor.css

```diff
@@ -400,7 +400,7 @@ img[src="/static/assets/7bd7487b704797cb5ab3cb441486ea70.svg"],
   background-color: var(--editorDarkMode-primary-transparent35);
 }
 [class*="modal_modal-overlay_"] {
-  background-color: var(--editorDarkMode-primary-overlay);
+  background-color: var(--editorDarkMode-popup);
```

Comment:
`primary-overlay` isn't used anywhere else, so you can remove its definition from `addon.json`.